Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.10
+// swift-tools-version: 6.0
```

Comment:
We may need to provide two Package manifest files (one for 5.10 and one for 6.0)